```

mobile-wiki/templates/components/post-search-results.hbs

  32:6  error  Incorrect indentation for `{{#each}}` beginning at L32:C6. Expected `{{#each}}` to be at an indentation of 8 but was found at 6.  block-indentation
```

--- a/app/templates/components/post-search-results.hbs
+++ b/app/templates/components/post-search-results.hbs
@@ -33,7 +33,7 @@
           {{#if (equal post.type "affiliate")}}
             {{post-search-results-affiliate-item affiliateUnit=smallAffiliateUnit postIndex=index}}
           {{else}}
-           {{post-search-results-item post=post postIndex=index}}
+            {{post-search-results-item post=post postIndex=index}}
           {{/if}}
         {{else}}
           <div class="post-search-results__no-posts">

--- a/app/templates/components/post-search-results.hbs
+++ b/app/templates/components/post-search-results.hbs
@@ -29,17 +29,17 @@
       />
     {{else}}
       <div class="post-search-results__items">
-      {{#each posts as |post index|}}
-        {{#if (equal post.type "affiliate")}}
-          {{post-search-results-affiliate-item affiliateUnit=smallAffiliateUnit postIndex=index}}
+        {{#each posts as |post index|}}
+          {{#if (equal post.type "affiliate")}}
+            {{post-search-results-affiliate-item affiliateUnit=smallAffiliateUnit postIndex=index}}
+          {{else}}
+           {{post-search-results-item post=post postIndex=index}}
+          {{/if}}
         {{else}}
-          {{post-search-results-item post=post postIndex=index}}
-        {{/if}}
-      {{else}}
-        <div class="post-search-results__no-posts">
-          {{{i18n "main.search-post-item-no-posts" ns="search"}}}
-        </div>
-      {{/each}}
+          <div class="post-search-results__no-posts">
+            {{{i18n "main.search-post-item-no-posts" ns="search"}}}
+          </div>
+        {{/each}}
       </div>
     {{/if}}
   </aside>

